pkg/controllers/job/plugins/ssh/ssh.go

```diff
@@ -148,7 +148,7 @@ func (sp *sshPlugin) mountRsaKey(pod *v1.Pod, job *batch.Job) {
 	}
 
 	if sp.sshKeyFilePath != SSHAbsolutePath {
-		var noRootMode int32 = 0755
+		var noRootMode int32 = 0600
```

Comment:
have you tested for that?

Reply:
![image](https://user-images.githubusercontent.com/13084946/129906186-38c0cda0-0609-4326-a193-46d1bac1117c.png)